src/pages/publication/references_tab/BookReferenceForm.tsx

```diff
@@ -51,14 +51,13 @@ const BookReferenceForm: FC = () => {
       <Field name={BookFieldNames.TYPE}>
         {({ field }: any) => (
           <FormControl variant="outlined" fullWidth>
-            <InputLabel>{t('common:type')}</InputLabel>
-            <Select {...field}>
+            <TextField select variant="outlined" label={t('common:type')} {...field}>
               {bookTypes.map(type => (
                 <MenuItem value={type.value} key={type.value}>
                   {t(type.label)}
                 </MenuItem>
               ))}
-            </Select>
+            </TextField>
           </FormControl>
```

Comment:
trenger vi FormControl i det hele tatt? er det bare for å sette fullWidth?

Reply:
nei, kan flytte fullWidth til TextField 👍